README.md

```diff
@@ -8,10 +8,10 @@
 [![David](https://david-dm.org/henryboldi/felony.svg)]()
 
 ## Download Felony
-You can download compiled versions of Felony for Windows, Mac OS X, and Linux from https://github.com/henryboldi/felony/releases. The app is currently in its pre-release stage, so it hasn't been fully tested on all platforms. Confirmed to be working on Mac, as that's what the developers own.
+You can download compiled versions of Felony for Windows, Mac OS X, and Linux from https://github.com/henryboldi/felony/releases. The app is currently in it's pre-release stage, so it hasn't been fully tested on all platforms. Confirmed to be working on Mac, as that's what the developers own.
```

Comment:
`its` is the correct form here, `it's` means `it is` while `its` is the possessive form of `it`.